src/Controller/OcrController.php

```diff
@@ -184,7 +185,7 @@ public function homeAction(): Response
      * @OA\Parameter(
      *     name="engine",
      *     in="query",
-     *     description="The engine to use, either `tesseract` or `google`.",
+     *     description="The engine to use, either `tesseract` or `google` or transkribus.",
```

Comment:
I think it should be changed to \`transkribus`, so that it conforms to the rest of the engine name styles